docs/_docs/test.md

```diff
@@ -3,153 +3,120 @@ nav_order: 6
 
+```
+mqtt test
+```
+
+By default, the test command will use MQTT 3 clients to test the broker first and will afterward check the connect
+restrictions returned by a connect of a MQTT 5 client. You can alter this behavior by specifying different
+[options](#options) when using the command.
 
-By default the test command will use MQTT 3 clients to test the broker first and will afterwards check the connect 
-restrictions returned by a connect of a MQTT 5 client. You can alter this behavior by specifying different 
-[options](#test-options) when using the command.
- 
-## Example   
+## Example
 
 ``` 
 $ mqtt test -h broker.hivemq.com
 MQTT 3: OK
-	- Maximum topic length: 65535 bytes
-	- QoS 0: Received 10/10 publishes in 47,59ms
-	- QoS 1: Received 10/10 publishes in 68,87ms
-	- QoS 2: Received 10/10 publishes in 43,18ms
-	- Retain: OK
-	- Wildcard subscriptions: OK
-	- Shared subscriptions: OK
-	- Payload size: >= 100000 bytes
-	- Maximum client id length: 65535 bytes
-	- Unsupported Ascii Chars: ALL SUPPORTED
+    - Maximum topic length: 65535 bytes
+    - QoS 0: Received 10/10 publishes in 47,59ms
+    - QoS 1: Received 10/10 publishes in 68,87ms
+    - QoS 2: Received 10/10 publishes in 43,18ms
+    - Retain: OK
+    - Wildcard subscriptions: OK
+    - Shared subscriptions: OK
+    - Payload size: >= 100000 bytes
+    - Maximum client id length: 65535 bytes
+    - Unsupported Ascii Chars: ALL SUPPORTED
 MQTT 5: OK
-	- Connect restrictions: 
-		> Retain: OK
-		> Wildcard subscriptions: OK
-		> Shared subscriptions: OK
-		> Subscription identifiers: OK
-		> Maximum QoS: 2
-		> Receive maximum: 10
-		> Maximum packet size: 268435460 bytes
-		> Topic alias maximum: 5
-		> Session expiry interval: Client-based
-		> Server keep alive: Client-based
+    - Connect restrictions: 
+        > Retain: OK
+        > Wildcard subscriptions: OK
+        > Shared subscriptions: OK
+        > Subscription identifiers: OK
+        > Maximum QoS: 2
+        > Receive maximum: 10
+        > Maximum packet size: 268435460 bytes
+        > Topic alias maximum: 5
+        > Session expiry interval: Client-based
+        > Server keep alive: Client-based
 ```
 
-<!---
-See also 
-```
-mqtt test --help
-```
--->
+***
 
-*** 
+## Options
 
-## Synopsis
+| Option | Long Version    | Explanation                                                                    | Default                      |
+|--------|-----------------|--------------------------------------------------------------------------------|------------------------------|
+| `-V`   | `--mqttVersion` | The MQTT version to test the broker on.                                        | Both versions will be tested |
+| `-a`   | `--all`         | Perform all tests for all MQTT versions                                        | `false` (Only test MQTT 3)   |
+| `-t`   | `--timeOut`     | The time to wait for the broker to respond (in seconds).                       | `10s`                        |
```

Comment:
adding the `s` to 10s doesn't seem consistent (connect keep alive is 60, without s) or needed, but we may not want to change the behaviour at this point

Reply:
In code it its without the s. We can change it inside the documentation.